ntp.go

```diff
@@ -139,12 +144,23 @@ type Response struct {
 	Leap           LeapIndicator // server's leap second indicator; see RFC 5905
 }
 
+type QOption struct {
+	Timeout time.Duration // defaults to defaultTimeout
+	Version int           // NTP protocol version, defaults to 4
+	Port    int           // NTP Server port for UDPAddr.Port, defaults to 123
+	IpTTL   int           // IP TTL to use for outgoing UDP packets
```

Comment:
idiomatic go does not "camel case" acronyms.  So this should either be `IPTTL` or just `TTL`.  I prefer `TTL`.  The comment clarifies that it is for the IP protocol.